### PR TITLE
Scenario instead of settings

### PR DESCRIPTION
**Pull request by StevenClontz**
More information on why this is necessary...

### CODE REVIEW

File: source/calculus/source/03-AD/04.ptx [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <title>Activities</title>
     <remark>
     <p>
-      In many different settings,
+      In many different scenarios,
```
Comment (siwelwerd) [inline source/calculus/source/03-AD/04.ptx:16]:
```suggestion
      In many different situations,
```

What about "situations"?

File: source/calculus/source/03-AD/04.ptx [hunk 2]
```diff
@@ -13,7 +13,7 @@
     <title>Activities</title>
```
Comment (StevenClontz) [inline source/calculus/source/03-AD/04.ptx:13]:
```suggestion
    <title>Puzzles</title>
```

Comment (StevenClontz) [reply source/calculus/source/03-AD/04.ptx:13]: eh never mind

**Comment by siwelwerd** [comment]
This was for training purposes only, closing to declutter

**Pull request closed without merging**